docs/src/main/sphinx/release/release-406.md

```diff
@@ -0,0 +1,67 @@
+# Release 406 (6 Jan 2023)
+
+## General
+
+* Add support for exchange spooling on HDFS when fault-tolerant execution is
+  enabled. ({issue}`15160`)
+* Improve planner estimates for queries containing outer joins over a `topN`
```

Comment:
Should it really be code highlighted and formatted like that? Or should we use what is displayed in a query plan without code highlight?